BrainCloudClient/tests/TestFixtureBase.cs

```diff
@@ -36,7 +39,33 @@ public void Setup()
             Dictionary<string, string> secretMap = new Dictionary<string, string>();
             secretMap.Add(AppId, Secret);
             secretMap.Add(ChildAppId, ChildSecret);
-            _bc.InitWithApps(ServerUrl, AppId, secretMap, Version);
+
+            //Use check so that we only run this initialization test once per test suite, instead of 3 times per unit test
```

Comment:
Hum... _what_ ?

We don't need to retest the init for each unit tests. Should have it's own test case. Maybe inside TestWrapper or it's own class.

Reply:
I guess based on how the testfixturebase works, is that we don't have a reference to the ServerUrl, AppId, secretMap, Version of the ids text because its protected information, and we would need to initialize a TestFixtureBase Object in the test file which feels quite convoluted

Reply:
It's a little different than testing an auth call, its needs all of the app information re-entered for the app. Maybe i can just put in the info we'd expect from the ids text as hard coded

Reply:
Do a new test, once entering it and it's setup, call initialize on it. Then do a call. The call should fail normally, but not trigger the big bad issue we were having.